src/ontology/edits/oeo-physical.omn

```diff
@@ -581,7 +669,7 @@ Class: CarbonDioxideEmissionTrafficSector
         <http://purl.obolibrary.org/obo/IAO_0000115> "A carbon dioxide emission traffic sector quantity is a carbon dioxide emission quantity that is restricted to carbon dioxide from the traffic sector."@en
     
     SubClassOf: 
-        CarbonDioxideEmission
+        Sector
```

Comment:
`CarbonDioxideEmissionTrafficSector` is not a sector, but I am fine to shift this class **temporarily** to sector and deal with it in issue #170.